ch-algorithms/quantum-phase-estimation.ipynb

```diff
@@ -572,7 +573,7 @@
     }
    ],
    "source": [
-    "backend = IBMQ.get_backend('ibmq_20_tokyo')\n",
+    "backend = provider.get_backend('ibmq_20_tokyo')\n",
```

Comment:
Can you please replace this one with `ibmq_vigo`? `ibmq_20_tokyo` is a premium device and may not be available to all readers.